src/compile-gate.lisp

```diff
@@ -16,6 +16,8 @@
   (handler-bind (#+sbcl (sb-ext:compiler-note #'muffle-warning))
     (compile nil form)))
 
+(global-vars:define-global-var **dotimes-iterator** 'pdotimes)
```

Comment:
I take it the `lparallel:` prefix is no longer needed.

Reply:
it is, we have a homebrew `pdotimes` in utilities.lisp